draft-ietf-tls-dtls-connection-id.md

```diff
@@ -96,6 +99,11 @@ This document defines an extension to DTLS 1.2 to add a Connection ID (CID) to t
 DTLS record layer. The presence of the CID is negotiated via a DTLS
 extension.
 
+Adding a CID to the ciphertext record format presents an opportunity to make
+other changes to the record format.  In keeping with the best practices
+established by TLS 1.3, the type of the record is encrypted, and
+a mechanism provided for adding padding to obfuscate the plaintext length.
```

Comment:
Being not a native english speaker, I feel, a "is" is missing? 

> and a mechanism is provided for adding padding to obfuscate the plaintext length.

Reply:
Looks like a construction that in Italian grammar is called (literal translation) "elliptical proposition".  You can omit subject or predicate in a sentence if they are unambiguously implied by the context.

Reply:
Thanks :-)